client/src/Components/ItemsForms.jsx

```diff
@@ -0,0 +1,40 @@
+export function ItemForms({ numberOfItems, out }) {
+
+  const forms = [];
+  for (let i = 0; i < numberOfItems; i++) {
+    forms.push(
+      <li key={i} className='container-l list-group-item m-2'>
+        <div className='row g-3'>
+          <div className='col'>
+            <label className='form-label' htmlFor={`item${i}`}>{`Item #${i + 1}`}</label>
```

Comment:
If you wrap the `<label>` tag around your `<input>` element, you don't need the `htmlFor` and `id` fields. I think that makes for cleaner code.